catalog/camel-report-maven-plugin/src/main/java/org/apache/camel/maven/RouteCoverageMojo.java

```diff
@@ -319,8 +332,16 @@ public void execute() throws MojoExecutionException, MojoFailureException {
             }
         }
 
+        // compute and log overall coverage across routes
+        AtomicBoolean overallCoverageAboveThreshold = new AtomicBoolean();
+        String out = templateOverallCoverageData(coveredNodes.get(), totalNumberOfNodes, overallCoverageAboveThreshold);
+        getLog().info("Overall coverage summary:\n\n" + out);
+        getLog().info("");
```

Comment:
what's the purpose of this line?

Reply:
These lines show the overall test coverage in the same style as the pre-existing code already showed it on a per-route-basis. It allows to also set an overall threshold and then fail if the overall coverage is below that.
This looks like this:
```
[INFO] Overall coverage summary:

Coverage: 22 out of 23 (95.7% / threshold 95.0%)
Status: Success
```

This is useful if you have dozens of routes with some only doing some minor debug logging or similar trivial things. Before, you could only set a threshold on a per-route-basis, e.g. all routes need to be covered at least 90%. At least for us, that would be very tedious, as we would have to cover each of the more trivial routes fully instead of setting an overall threshold.

Reply:
I meant this specific line: `getLog().info("");`

Reply:
Ah, that is just an empty line. I wanted to be consistent with lines 274 and 326. There is an empty line like this after each of the per-route coverage summaries.